tests/sampleTests/replaySample.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { expect, test } from '@playwright/test';
+import { Polly } from '@pollyjs/core';
+
+import { submitPaymentResult } from '../api.mock';
+import { replayInitializer } from '../polly.global.setup';
+
+// Uncommmnet to continue in a headed browser
+// test.use({
+//     headless: false,
+//     viewport: { width: 1000, height: 1000 },
+// });
+
+test.describe('Replay mode sample', () => {
+
+    let polly: Polly;
+
+    test.beforeEach(async ({ page }) => {
+        // 1. Setup PollyJS
+        polly = await replayInitializer({
+            playwrightContext: page,
+            recordingName: 'sample',
+            storeURL: 'https://my-dev-store-745516528.store.bcdev',
+        });
+
+        await page.goto('http://localhost:8080/');
+    });
+
+    test('Bigpay Test Payment Provider is working', async ({page}) => {
+
+        // 1. Setup PollyJS
+        // Since we put it into test.beforeEach(), don't have to initialize again
+        // const polly = await replayInitializer({
+        //     playwrightContext: page,
+        //     recordingName: 'test-recording',
+        //     storeURL: 'https://my-dev-store-745516528.store.bcdev',
+        // });
+
+        // 2. Serve additional static files with Playwright
+        await page.route('**/checkout/payment/hosted-field?**', route => route.fulfill( {status: 200, path: './tests/sampleTests/_support/hostedField.html' } ));
+
+        // 3. Mock API endpoints via Playwright
+        await page.route('**/api/public/v1/orders/payments', route => route.fulfill( {status: 200, contentType: 'application/json', body: JSON.stringify(submitPaymentResult)} ));
```

Comment:
Is there a reason why this API call needs to be mocked manually?

Reply:
Its response is always a 405 error during the replay. 
So I manually set it to 200. There might be a better way. I'll try it later.